tests/baselines/reference/user/lodash.log

```diff
@@ -126,6 +129,8 @@ node_modules/lodash/_createWrap.js(96,26): error TS2345: Argument of type 'Timer
 node_modules/lodash/_createWrap.js(97,100): error TS2532: Object is possibly 'undefined'.
 node_modules/lodash/_createWrap.js(98,28): error TS2345: Argument of type 'TimerHandler' is not assignable to parameter of type 'Function'.
   Type 'string' is not assignable to type 'Function'.
+node_modules/lodash/_createWrap.js(100,44): error TS2345: Argument of type 'any[]' is not assignable to parameter of type '[TimerHandler, number, any?, (any[] | undefined)?, (any[] | undefined)?, (any[] | undefined)?, (any[] | undefined)?, (any[] | undefined)?, (number | undefined)?, (number | undefined)?]'.
```

Comment:
since we don't create tuples, this `apply` call won't work with a type annotation on the argument's declaration. Unfortunate.